Library/Homebrew/software_spec.rb

```diff
@@ -122,9 +122,12 @@ def option(name, description = "")
         puts "Symbols are reserved for future use, please pass a string instead"
         name = name.to_s
       end
+      unless String === name
```

Comment:
I also think that `is_a?` would be preferable and one step towards making `brew style` actually useful on our code base. That said, I can see the motivation for using `===` in this case.